lib/go/test/flow_idtable_staking_test.go

```diff
@@ -2942,27 +3132,27 @@ func TestIDTableStaking(t *testing.T) {
 
 	})
 
-	// End staking auction and move tokens in the same transaction
-	t.Run("Should end staking auction and move tokens in the same transaction", func(t *testing.T) {
+	// // End staking auction and move tokens in the same transaction
```

Comment:
Any reason this is commented out?

Reply:
because I updated the transaction after it to do the same thing. I'll remove the commented out lines.